custom_components/bdr_thermostat/config_schema.py

```diff
@@ -10,12 +10,15 @@
 )
 
 CONF_PAIR_CODE = "pairing_code"
+CONF_BRAND = "brand"
 
 
 SUPPORT_FLAGS = SUPPORT_TARGET_TEMPERATURE | SUPPORT_PRESET_MODE
 CLIMATE_SCHEMA = {
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Required(CONF_USERNAME): cv.string,
     vol.Required(CONF_PASSWORD): cv.string,
-    vol.Required(CONF_PAIR_CODE): cv.string,
```

Comment:
Did you mean to remove this line or was that a mistake when adding the new option to configure the brand?

Reply:
That was not itentional indeed. The paringcode is required for this to work. It probably works for me because i had it configured before. I had it disabled during testing.